src/main/java/frc/robot/maps/subsystems/ArmMap.java

```diff
@@ -0,0 +1,64 @@
+package frc.robot.maps.subsystems;
+
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+import com.chopshop166.chopshoplib.motors.SmartMotorController;
+
+import edu.wpi.first.math.controller.PIDController;
+
+public class ArmMap {
+    public ArmMap() {
+    }
+
+    public PIDController pid = new PIDController(0, 0, 0);
```

Comment:
These member variables should be set in the constructors, instead of as part of the class members